persistence/src/main/scala/hmda/persistence/processing/HmdaFileParser.scala

```diff
@@ -60,12 +61,16 @@ class HmdaFileParser(submissionId: SubmissionId) extends HmdaPersistentActor wit
 
     case ReadHmdaRawFile(persistenceId) =>
       publishEvent(ParsingStarted(submissionId))
+      var encounteredParsingErrors: Boolean = false
```

Comment:
Using this boolean variable, `encounteredParsingErrors`, seemed like the easiest/most efficient way to decide whether or not the file had any parsing errors. 

Is it bad form to use a `var`? If so, what would be better?